pytd/writer.py

```diff
@@ -265,14 +277,73 @@ def write_dataframe(self, dataframe, table, if_exists, fmt="csv"):
             - msgpack
                 Convert to temporary msgpack.gz file. Fast option but might be
                 unstable because of skipping ``tdclient.API._prepare_file()``
+
+        keep_list : boolean, default: False
+            If this argument is True, keep list or numpy.ndarray column as list, which
+            will be converted array<T> on Treasure Data table.
+            Each type of element of list will be converted by
+            ``numpy.array(your_list).tolist()``.
+
+            If True, ``fmt`` argument will be overwritten with ``msgpack``.
+
+            Examples
+            ---------
+
+            A dataframe containing list will be treated array<T> in TD.
+
+            >>> import pytd
+            >>> import numpy as np
+            >>> import pandas as pd
+            >>> df = pd.DataFrame(
+            ...     {
+            ...         "a": [[1, 2, 3], [2, 3, 4]],
+            ...         "b": [[0, None, 2], [2, 3, 4]],
+            ...         "c": np.array([1, np.nan, 3], np.array([2, 3, 4]))
```

Comment:
This unfortunately doesn't work..

```py
>>> df = pd.DataFrame(
...     {
...         "a": [[1, 2, 3], [2, 3, 4]],
...         "b": [[0, None, 2], [2, 3, 4]],
...         "c": np.array([1, np.nan, 3], np.array([2, 3, 4]))
...     }
... )
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
TypeError: data type not understood
```

```suggestion
            ...         "c": [[1, np.nan, 3], np.array([2, 3, 4])]
```

Reply:
😱 b9c7604